examples/experimental/i3s-17-and-debug/constants.js

```diff
@@ -37,3 +37,10 @@ export const BOUNDING_VOLUME_WARNING_TYPE = 'boundingVolume';
 export const LOD_WARNING_TYPE = 'lod';
 export const PARENT_LOD_WARNING_TYPE = 'parentLod';
 export const GEMETRY_VS_TEXTURE_WARNING_TYPE = 'geometryVsTextures';
+
+export const BOUNDING_VOLUME_SHAPE_MODE = {
```

Comment:
When are these constants used? If presented to the user, should we use the terminology `Bounding Sphere` and `Oriented Bounding Box`?

Reply:
`BOUNDING_VOLUME_TYPE` ?

Reply:
it's now fixed with names of constants